extensions/jmx_metrics/README.md

```diff
@@ -0,0 +1,116 @@
+OpenTelemetry Extensions JMX Metric Gatherer
+============================================
+
+[![Javadocs][javadoc-image]][javadoc-url]
+
+* Java 7 compatible.
+
+[javadoc-image]: https://www.javadoc.io/badge/io.opentelemetry/opentelemetry-contrib-runtime-metrics.svg
+[javadoc-url]: https://www.javadoc.io/doc/io.opentelemetry/opentelemetry-contrib-runtime-metrics
+
+This utility provides an easy framework for gathering and reporting metrics based on queried
+MBeans from a JMX server.  It loads a custom Groovy script establishes a helpful bound `otel`
+object with methods for obtaining MBeans and constructing synchronous OpenTelemetry instruments:
+
+### Usage
+
+```bash
+$ java -cp <opentelemetry-java jars> io.opentelemetry.extensions.metrics.jmx.JmxMetrics -config ./config.json
+```
+
+##### `config.json` example
```

Comment:
Would it be better to use Java properties and allow them to be specified on the command line? It may be easier for some cases instead of having to generate a file? e.g. `java -jar whatever.jar -Dotel.intervalSeconds=5`. Could also use properties file instead of having to include dependencies.